rules/prefer-math-min-max.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const MESSAGE_ID = 'prefer-math-min-max';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `{{replacement}}` to simplify ternary expressions.',
+};
+
+/**
+@param {import('eslint').Rule.RuleContext} context
+@param {import('estree').ConditionalExpression} node
+@param {import('estree').Node} left
+@param {import('estree').Node} right
+@param {string} method
+*/
+function reportPreferMathMinOrMax(context, node, left, right, method) {
+	const {sourceCode} = context;
+
+	return {
+		node,
+		messageId: MESSAGE_ID,
+		data: {
+			replacement: `${method}()`,
+		},
+		* fix(fixer) {
+			/**
+			 * ```js
+			 * function a() {
+			 *   return+foo > 10 ? 10 : +foo
+			 * }
+			 * ```
+			 */
+			if (node.parent.type === 'ReturnStatement' && node.parent.argument === node && node.parent.start + 'return'.length === node.start) {
```

Comment:
Use this instead https://github.com/sindresorhus/eslint-plugin-unicorn/blob/1e367bbe6f9e5a135fd7476b18df714e26928c4a/rules/fix/fix-space-around-keywords.js#L12

Reply:
f5afe1e12a478305043c09b3e6fcb473aeec070e